src/ServiceManagement/Services/Commands.Utilities/ChangeLog.md

```diff
@@ -19,4 +19,5 @@
 -->
 ## Current Release
 
-## Version 3.1.0
\ No newline at end of file
+## Version 3.1.0
+* Adding changes related to communication of retirement message for managed cache customers. For more information, see http://go.microsoft.com/fwlink/?LinkID=717458
```

Comment:
This should go in the 'current release' section, Version 3.1.0 has already shipped